src/button/default.scss

```diff
@@ -10,7 +10,7 @@
     color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
   }
 
-  > .iui-icon {
+  .iui-icon {
```

Comment:
Wont we have issues with this one like we already had in other places where rule was applied to deeper icons, where it was not needed?